docs/manual/figures/poly-hierarchy.svg

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The file name should include determinism in the name.  Also the top two boxes are cut off. (https://github.com/t-rasmud/checker-framework/blob/bf4429bec883f6b50e2bef0373b21f37334d8cc3/docs/manual/figures/poly-hierarchy.svg)